checkpoints/check_audio.py

```diff
@@ -59,7 +59,8 @@ def get_sound_cards():
 
 def is_service_active():
     """Returns True if the voice-recognizer service is active."""
-    output = subprocess.check_output(['systemctl', 'show', SERVICE_NAME]).decode('utf-8')
+    output = subprocess.check_output(['systemctl', 'show',
```

Comment:
Thanks. This formatting thing has been bugging me quite a bit. I've reverted all formatting changes.